src/visualizers/panels/PlotlyGraph/PlotlyGraphControl.js

```diff
@@ -71,7 +72,9 @@ define([
             hasGraph = !!graphNode;
         }
         if(hasGraph){
-            desc = this.figureExtractor.extract(graphNode);
+            const graphInfo = this.figureExtractor.GMENodeToMetadataJSON(graphNode);
```

Comment:
If the code is updated to call `toJSON` (`GMENodeToMetadataJSON`) internally, this segment of code wouldn't need to change and could still just call `extract`.